### PR TITLE
Image Snapshot Tests

### PR DESCRIPTION
**Pull request by notjosh**
The comments on #8 made me think more about image snapshot tests, to make sure we're not accidentally causing regressions in the HTML->image chain(s). Since #8 changes the renderer, it's probably a good idea to get these tests in sooner rather than later.

There are only a couple of simple tests for now, but we can add cases when we find issues, and it's enough (imo!) to catch any obvious issues.

Couple of things:

- because of fonts/font rendering, we should only run/care about test results from Docker environment for consistency
- added a `make ci-snapshot-update` task to bump the snapshots
- mounted the entire repo as a volume in the dev/test Docker environments (finally!) for quicker feedback loops

### CODE REVIEW

**Comment by notjosh** [comment]
Sidebar: would be nice to get the CI status inline here. I think that means bumping some API keys for the integration? Been a while since I've set it up.

Nevertheless, tests are green here: https://circleci.com/gh/notjosh/sirius/5

File: sirius/coding/templating.py [hunk 1]
```diff
@@ -8,13 +8,13 @@
 # TODO apply scrubber library if wanted.
 # https://pypi.python.org/pypi/scrubber
 
-def default_template(raw_html, from_name):
+def default_template(raw_html, from_name, date=datetime.datetime.now()):
```
Comment (joerick) [inline sirius/coding/templating.py]: `datetime.datetime.now()` will be evaluated only once, when the function is defined. Better to assign `None` as a default and test `if date is None:` in the function body to set a default

**Review by joerick — CHANGES_REQUESTED** [review]
Nice one. I'm interested to see how this turns out.

Just one thing needs fixing here, I think.

BTW, I added PR test runs to Circle, was just a missing checkbox on the settings.

File: sirius/coding/templating.py (hunk 1 above)
Comment (notjosh) [reply sirius/coding/templating.py]: Huh, I didn't know that! Good catch, and noted for future. Fixed in 044be3bf080b540fdba318d203edd6a327e56e82

**Comment by notjosh** [comment]
> BTW, I added PR test runs to Circle, was just a missing checkbox on the settings.

🎉

**Review by joerick — APPROVED** [review]
Sorry for the delay, been busy! Looks good, merge whenever you're ready :)

**Comment by joerick** [comment]
I could use faster tests, so I'm gonna merge :)

**Pull request merged**